meta_error_test.go

```diff
@@ -0,0 +1,94 @@
+package microerror
+
+import (
+	"testing"
+)
+
+func Test_MetaError(t *testing.T) {
```

Comment:
Maybe some comments are missing here but the basics as described in the comments are verified and work.